x-pack/plugins/security_solution/public/cases/components/add_comment/index.tsx

```diff
@@ -11,14 +11,14 @@ import styled from 'styled-components';
 import { CommentRequest } from '../../../../../case/common/api';
 import { usePostComment } from '../../containers/use_post_comment';
 import { Case } from '../../containers/types';
-import { MarkdownEditorForm } from '../../../common/components/markdown_editor/form';
+import { MarkdownEditorForm } from '../../../common/components/markdown_editor/new_form';
 import { InsertTimelinePopover } from '../../../timelines/components/timeline/insert_timeline_popover';
 import { useInsertTimeline } from '../../../timelines/components/timeline/insert_timeline_popover/use_insert_timeline';
 import { Form, useForm, UseField } from '../../../shared_imports';
 
 import * as i18n from './translations';
 import { schema } from './schema';
-import { useTimelineClick } from '../utils/use_timeline_click';
+import { useTimelineClick } from '../../../common/containers/use_timeline_click';
```

Comment:
nit -> what do you think if we put this hook in this folder instead `x-pack/plugins/security_solution/public/common/utils/timeline/`, it seems to be a better home for it